lib/call-session.js

```diff
@@ -861,15 +861,17 @@ Duration=${payload.duration} `
         }
       }
       else {
-        const response = await dlg.other.request({
-          method: 'INFO',
-          headers: req.headers,
-          body: req.body
-        });
-        res.send(response.status, {
-          headers: response.headers,
-          body: response.body
+        const immutableHdrs = ['via', 'from', 'to', 'call-id', 'cseq', 'max-forwards', 'content-length'];
```

Comment:
Why is this needed?  What happens when you just send all the headers? drachtio server should itself remove those headers I thought